docs/pages/reference/cli/tctl.mdx

```diff
@@ -2051,6 +2051,95 @@ prod       Done  2025-01-15 18:00:12 outside_window
 backup     Done  2025-01-14 20:00:31 outside_window
 ```
 
+## tctl plugins install awsic
+
+Install the AWS Identity Center integration.
+
+```code
+$ tctl plugins install awsic <flags>
+```
+
+### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--access-list-default-owner` | none | Teleport username | A Teleport user to set as the default owner for the imported Access Lists. Multiple flags allowed. |
+| `--scim-url` | none | URL | AWS Identity Center SCIM provisioning endpoint. Must be a valid AWS Identity Center SCIM endpoint URL, as per [this AWS Knowledge Center article](https://repost.aws/knowledge-center/iam-identity-center-provision). |
+| `--[no]force-scim-url` |  `--no-force-scim-url` | none | Force the use provided SCIM provisioning endpoint even if it fails SCIM endpoint validation. |
+| `--scim-token` | none | text | The AWS Identify Center SCIM provisioning access bearer token. |
+| `--instance-arn` | none | ARN | The ARN of the Identity Center instance to manage |
+| `--instance-region` | none | text | The AWS region where the Identity Center instance is located. |
+| `--[no-]use-system-credentials` | `--no-use-system-credentials` | none | Use system AWS credentials to authenticate with AWS. Note that currently only `--use-system-credentials` is supported by `tctl`. |
```

Comment:
> Note that currently only `--use-system-credentials` is supported by `tctl`.

This is a little confusing. Does this mean to say that "tctl" supports only use-system-creds, or that use-system-creds is only supported by tctl and not web UI currently? I assume it's the latter, in which case it feels like the word "only" should move.

```suggestion
| `--[no-]use-system-credentials` | `--no-use-system-credentials` | none | Use system AWS credentials to authenticate with AWS. Note that currently `--use-system-credentials` is only supported by `tctl`. |
```

Reply:
Technically, both